File: apps/ios/AboAlAbed/DesignSystem/BrandTheme.swift
import SwiftUI

enum BrandTheme {
    static let brand = Color(red: 0.804, green: 0.125, blue: 0.149)
    static let brandDeep = Color(red: 0.545, green: 0.067, blue: 0.086)
    static let cream = Color(red: 1.0, green: 0.969, blue: 0.937)
    static let sand = Color(red: 0.969, green: 0.894, blue: 0.78)
    static let sun = Color(red: 1.0, green: 0.722, blue: 0.282)
    static let ink = Color(red: 0.11, green: 0.11, blue: 0.11)
    static let success = Color(red: 0.12, green: 0.48, blue: 0.30)
    static let warning = Color(red: 0.82, green: 0.50, blue: 0.09)

    static let heroGradient = LinearGradient(
        colors: [brandDeep, brand, sun],
        startPoint: .topLeading,
        endPoint: .bottomTrailing
    )

    static let panelGradient = LinearGradient(
        colors: [Color.white.opacity(0.96), cream],
        startPoint: .top,
        endPoint: .bottom
    )
}

File: apps/android/app/src/main/java/com/aboalabed/uae/core/design/BrandTheme.kt
package com.aboalabed.uae.core.design

import androidx.compose.material3.MaterialTheme
import androidx.compose.material3.lightColorScheme
import androidx.compose.runtime.Composable
import androidx.compose.ui.graphics.Brush
import androidx.compose.ui.graphics.Color

object BrandColors {
    val Brand = Color(0xFFCD2026)
    val BrandDeep = Color(0xFF8B1116)
    val Cream = Color(0xFFFFF7EF)
    val Sand = Color(0xFFF7E4C7)
    val Sun = Color(0xFFFFB848)
    val Ink = Color(0xFF1C1C1C)
    val Success = Color(0xFF1F7A4D)
    val Warning = Color(0xFFD18017)

    val HeroGradient = Brush.linearGradient(listOf(BrandDeep, Brand, Sun))
    val PanelGradient = Brush.verticalGradient(listOf(Color.White, Cream))
}

@Composable
fun AboAlAbedTheme(content: @Composable () -> Unit) {
    MaterialTheme(
        colorScheme = lightColorScheme(
            primary = BrandColors.Brand,
            onPrimary = Color.White,
            secondary = BrandColors.Sun,
            background = BrandColors.Cream,
            surface = Color.White,
            onSurface = BrandColors.Ink,
            error = BrandColors.BrandDeep
        ),
        content = content
    )
}
